tests/function/test_config.py

```diff
@@ -144,25 +146,12 @@ def test_print_in_virtwho_conf(self, virtwho, globalconf, hypervisor_data):
         guest_id = hypervisor_data["guest_uuid"]
         globalconf.update("global", "print_", "True")
         globalconf.update("global", "debug", "True")
-        result = virtwho.run_service()
-        assert (
-            result["error"] == 0
-            and result["send"] == 0
-            and result["thread"] == 0
-            and result["debug"] is True
-            and guest_id in result["log"]
-        )
+        _, output = ssh_host.runcmd("virt-who", if_stdout=True)
```

Comment:
could not use the runserver for print function, which will not print any mapping in rhsm log.